server.js

```diff
@@ -31,21 +30,19 @@ mongoose
   .then(() => console.log("Now connected to MongoDB!"))
   .catch(err => console.error("Something went wrong", err));
 
-// ... other app.use middleware
+// Load index.html page provided
 app.use(express.static(path.join(__dirname, "client", "build")));
 
-// parse request body as json
+// Parse request body as json
 app.use(express.json());
+// Custom Routing for our app here
+app.use("/api/user", user);
 
-// using custom route
-app.use("/api/users", users);
 app.use("/api/auth", auth);
+app.use('/api/profile', require('./routes/profile'));
```

Comment:
Put the require function at the top of the file like the others

Reply:
resolved